PULL_REQUEST_TEMPLATE.md

```diff
@@ -23,7 +23,7 @@ before and after the PR.-->
 - [ ] All tests passed (See [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
 - [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Aignitionrobotics+repo%3Aosrf%2Fsdformat+archived%3Afalse+) to support the maintainers
 
-**Note to maintainers**: Remember to use **Squash-Merge**
+**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary.
```

Comment:
Maybe add a reminder to keep DCO sign offs if editing the message. Otherwise, we'll get DCO failures when merging forward.

Reply:
good call

Reply:
how does  https://github.com/ignitionrobotics/.github/pull/4/commits/f08d22e9450c9d57934fb13aa5c9fe3c93f1f2bf look to you?